src/utils.ts

```diff
@@ -298,3 +301,33 @@ export class UDC {
         return UDC.instance;
     }
 }
+
+export function readContract(contractPath: string) {
+    const parsedContract = parse(
+        fs.readFileSync(contractPath).toString("ascii")
+    ) as CompiledContract;
+    return {
+        ...parsedContract,
+        program: compressProgram(parsedContract.program)
+    };
+}
+export function jsonToBigInt(alwaysParseAsBig: boolean) {
+    return JsonBigint({
+        alwaysParseAsBig,
+        useNativeBigInt: true,
+        protoAction: "preserve",
+        constructorAction: "preserve"
+    });
+}
+
+const { parse } = jsonToBigInt(false);
+
+export function bnToDecimalStringArray(rawCalldata: bigint[]) {
+    return rawCalldata.map((x) => x.toString(10));
+}
+
+export function estimatedFeeToMaxFee(amount?: bigint, maxFee?: Numeric, overhead = 0.5) {
```

Comment:
Unneeded parameter maxFee. It's only purpose here is to be overwritten.

Reply:
https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/293/commits/672522ac4a38bc9e70bdfd80a54a6fd9852da162 Great catch!